src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/dq/DirectionsQuestionnaireGenerator.java

```diff
@@ -112,7 +112,7 @@ private DocmosisTemplates getDocmosisTemplate(CaseData caseData) {
                     templateId = N181_CLAIMANT_MULTIPARTY_DIFF_SOLICITOR;
                 }
                 break;
-                //FALL-THROUGH
+            //FALL-THROUGH
```

Comment:
i think this is remnant code that doesn't need to be there anymore, can be deleted pls

Reply:
Done